frontend/src/components/main-view/details/product-details/ProductDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {useAppSelector} from "../../../../app/hooks";
+import {selectCurrentProduct} from "../../../../slicer/productSlice";
+import { Divider, Grid, Typography} from "@mui/material";
+import {parseName} from "../../../../interfaces/IThumbnailData";
+
+//component imports
+
+//interface imports
```

Comment:
```suggestion
//interface imports
```

Reply:
all these comments get autogenerated by the script that generates my components. While obviously being redundant in this case, I tend to leave them in because you never know if in the future there might be some imports added. They help me alot to keep my imports organized